events/events-intro.md

```diff
@@ -0,0 +1,67 @@
+![Interaction paths between DiSSCo technical infrastructure, CMSes and additional research or application systems](https://github.com/DiSSCo/openDS/blob/jbstatgen_events/events/FullPath_Step3_4_20221006_half.drawio.png "AbstractionLayer_png")
+
+**Figure 2**: The abstraction layer mediating the mapping between the data model of the CMS and the DOA with its openDOs.
+
+### Provenance Ontology as Standard for Events
+
+Event-based transactions between CMS and DiSSCo DOA will follow the model provided by the W3C-recommended Provenance Ontology (PROV). The model has three main elements: an entity (the openDS), an activity (a predefined operation to be conducted on the entity) and an agent, which might act in a specific role. In the interaction between CMS and DiSSCo DOA there are a transmitting and a receiving partner as agents.
+
+The details of event-based transactions are still under development. Questions that need to be decided are, for example:
+
+* *Will events always involve fully wrapped digital objects, that is openDS entities, or will only the information be transmitted that is impacted by a change? This involves considerations of suitable granularity.*
+* *Associated with the previous question are also questions about which types of operations are needed or performed? Will* diffs *, that is checks for differences, be sufficient or are more specialized and thus explicit operation types necessary? Such specific types can refer specifically to, e.g., updates of taxon identifications, changes in locations during loans, etc.*
+* *Will the transmitting partner first only send a notification informing about a change and the receiving partner then decides if they are interested in the change and its associated transaction? If the receiving partner is interested, they request the full information about the change, that is, the "payload" of the change. Alternatively, the transmitting partner might send the payload of the change already with the notification and the receiving partner, if interested can immediately incorporate the change in its pool of digital objects or records.*
+
+### openDigitalEvents
+
+The event-based transactions become open FAIR digital objects (openFDOs) themselves in the form of **openDigitalEvents** (openDEs). For this purpose, events are associated with a predefined set of metadata that is sufficiently informative for the event as openDE to "stand on its own". Thus, openDEs are wrapped in several layers of (meta)data, which contain and store all context necessary to understand (and/or recreate) the event and hence enable the digital object to be self-explanatory, self-sufficient and fully functional. At the same time, openDEs and their metadata layers are expected to be minimal, only providing functions and information that are indispensable for the transmission and the generation of the required action in the receiving system. Hence they should not carry excessive functions and information.
```

Comment:
Not certain of this. The current idea was to not give each version a PID but add the version to the PID. So if `20.5000.1025/LP6-KEG-B1T` is the specimen than the event record of the first version is `20.5000.1025/LP6-KEG-B1T/1`. If we give each event a unique PID than the amount would grow massively.

Reply:
Mmh, I have been wondering about that, too.

On the other hand, the openDEvent would be basically only the information of the entry that is logged in the provenance record for an event. The provenance record in a way is a record and repository of events. 

How are we actually able to identify the events needed to go back to a previous state of an openDS (reproducibility)? How do we navigate the logs?